### DISCUSSION TITLE
Not Found for https://github.com/SIRAHFF/documentation/releases/download/AMBER/sirah_x2.3_23-11.amber.tar.gz

### DISCUSSION THREAD

**Discussion by xih72** [SIRAHFF/documentation#73] [Q&A] [answered]
After clicking "SIRAH for Amber" from the webpage of Sirah tutorial for Amber, it goes to
https://github.com/SIRAHFF/documentation/releases/download/AMBER/sirah_x2.3_23-11.amber.tar.gz
but the result is "Not Found".

**Comment by SIRAHFF** [comment, marked as answer]
When we changed the .tar.gz, the link stopped working. 
It should work now.
